mava/configs/env/rware.yaml

```diff
@@ -4,3 +4,4 @@ defaults:
   - rware_scenario: tiny-2ag # [tiny-2ag, tiny-4ag, tiny-4ag-easy, small-4ag]
 
 env_name: RobotWarehouse-v0
+task_name: tiny-2ag
```

Comment:
I think it is better to leave this in the child configs, otherwise we always have to set the scenario name and the task_name which could lead to accidental mismatches.